### DISCUSSION TITLE
Minor background mismatch with shell variable navigation ZSH (OMZ)

### DISCUSSION THREAD

**Discussion by OlaoluwaM** [Eugeny/tabby#8422] [Q&A]
![image](https://github.com/Eugeny/tabby/assets/37044906/922f100a-f71a-4c33-ba49-d824c61003c3)

Notice how the background surrounding the text `$DOTS` is darker than the surrounding background color? Any way to fix this? Not sure if it is caused by tabby specifically, but I haven't experienced this issue with other terminal emulators, so I figured I should ask here. Thanks.

If this is better categorized as an issue, let me know and I'll move it over.

**Comment by OlaoluwaM** [comment]
Also happens when I switch to a light color scheme:

![image](https://github.com/Eugeny/tabby/assets/37044906/a03b37b9-200f-4f8d-a5f8-77964f14f842)

**Comment by Memphizzz** [comment]
I have the same problem, I think this is the same as in: https://github.com/Eugeny/tabby/issues/1222.

Looking forward to a fix.
